frontends/web/src/style/variables.css

```diff
@@ -25,6 +25,8 @@
     --color-primary: var(--color-blue);
     --color-secondary: var(--color-gray);
     --color-light: var(--color-mediumgray);
```

Comment:
in the old PR I wrote that `--color-light` isn't used anywhere, could you double check and remove?

Reply:
We use `--color-mediumgray` a lot and for a fonts, backgrounds and borders. So we should replace `--color-mediumgray` with `--color-light` for fonts. For backgrounds, we should make a new variable that uses `--color-mediumgray`.